app/travel_processor/deploy-both/ui5-deploy-node.yaml

```diff
@@ -0,0 +1,56 @@
+# yaml-language-server: $schema=https://sap.github.io/ui5-tooling/schema/ui5.yaml.json
+specVersion: "2.6"
+metadata:
+  name: sap.fe.cap.travelN
+type: application
+
+resources:
+  configuration:
+    propertiesFileSourceEncoding: UTF-8
+builder:
+  resources:
+    excludes:
+      - "/test/**"
+      - "/localService/**"
+  customTasks:
+    - name: ui5-task-zipper
+      afterTask: generateCachebusterInfo
+      configuration:
+        archiveName: travel-processor-node
+    - name: custBuildTask-manifest
+      beforeTask: generateManifestBundle
+      configuration:
+        stack: node
+    - name: custBuildTask-compPreload
+      afterTask: generateComponentPreload
+      configuration:
+        stack: node
```

Comment:
Can we combine the `compPreload` and `manifest` tasks?  They do similar things, but their individual bootstrapping requires so much config...

Reply:
That's how I had it initially, but that doesn't work: between "custBuildTask-manifest" and "custBuildTask-compPreload" runs the standard task "generateComponentPreload", which needs as input the (modified) manifest.yaml.